cloud-azure/src/main/resources/templates/arm-v2-freeipa.ftl

```diff
@@ -97,7 +97,7 @@
              <#list igs as group>
              <#if ! securityGroups[group.name]?? || ! securityGroups[group.name]?has_content>
              {
-               "apiVersion": "2015-05-01-preview",
+               "apiVersion": "2016-11-01",
```

Comment:
what is the reason we can't use the same version for this and https://github.com/hortonworks/cloudbreak/pull/15072/files#diff-5a5ddcd654eb39cf2f08c485d02064303d338dee754858acdb0036437095f148R152 ?
Also isn't it possible to use a more recent version? I mean this is already almost 7 year old version

Reply:
I have tried to minimize the impact by only changing the apiVersion for resource which is needed to support zones. Also while updating version, I have updated it to minimum version of resource which supports it. I checked that there are multiple things which need to be updated in ARM Template by running it against toolkit. We should take this up as separate task.

Reply:
I think no one will ever update this. I think the best way to update to the right version when you already have to touch it